inst/private/check_and_convert.m

```diff
@@ -0,0 +1,36 @@
+function obj = check_and_convert()
+  newl = sprintf('\n');
+  
+  str_check_is_sym = strjoin({'if temp is None or isinstance(temp, (sp.Basic, sp.MatrixBase)):',
```

Comment:
`None` is a bit of special case: you can probably ignore for now, or see `@sym/isNone` for tests.  Basically, its not a SymPy object but it can get returned...